pkg/cmd/dev/io/os/os.go

```diff
@@ -212,10 +217,33 @@ func (o *OS) CopyFile(src, dst string) error {
 		if err != nil {
 			return err
 		}
-		dstFile, err := os.Create(dst)
+		defer func() { _ = srcFile.Close() }()
+		dstFile, err := os.Open(dst)
+		if err != nil && !os.IsNotExist(err) {
+			return err
+		} else if err == nil {
+			srcInfo, err := srcFile.Stat()
```

Comment:
I think we want a `defer func() { _ = dstFile.Close() }()` in this `if err == nil` block as well.